### ISSUE TITLE
Size of side panels on desktop

### ISSUE THREAD

**Issue by el-srabo** [umap-project/umap#1825]
**Describe the bug**
The side panels (and the text within) when clicking on a feature are really small compared to the rest of the UI, even when the browser zoom level is at 100%. 
I usually mostly visit my map on a phone but I seem to remember the text being of adequate size when using a desktop before the latest update.

**To Reproduce**
1. Go to https://framacarte.org/m/148987/
2. Click on a feature

**Expected behavior**
The text should be bigger and the panel itself could be a bit larger

**Screenshots**
![5](https://github.com/umap-project/umap/assets/150371715/4b5fcab3-5190-40b2-a685-e7e90a76e547)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Edge
 - Version: 124.0.2478.105

**Comment by dawansv** [comment]
I agree with this. One way to mitigate part of it is if there was an option to set the popup toggle size to expanded when opening the first time. Same for the main map caption.

**Comment by yohanboniface** [comment]
Related #2191
